linode/manager
# feat: [M3-7311] - Create Load Balancer flow - manage state

## Description 📝
Create Load Balancer flow - Manage state.

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Checkout the branch and run the app locally
- Enable MSW


### Verification steps 
(How to verify changes)
- Navigate to http://localhost:3000/loadbalancers
- Click on "Create Load Balancer" Button.
- Verify Load Balancer label field state.
- Verify Create LoadBalancer payload schema W.R.T API spec.
- Verify tests for LoadBalancerLabel.test.tsx.

## As an Author I have considered 🤔

*Check all that apply*
